pkg/health/client.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package health
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/pb"
+)
+
+type HealthClient struct {
```

Comment:
Client should be separated from the endpoint implementation otherwise compiling the client you'll end up pulling in all the service dependencies.

Reply:
done